libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/coordinator/InfoPanelCoordinator.kt

```diff
@@ -90,4 +104,30 @@ internal class InfoPanelCoordinator(
             setGuidelinePosition(bottomSheet)
         }
     }
+
+    /**
+     * An OnLayoutChangeListener that ensures the bottom sheet is always laid out at the bottom of
+     * the parent view when in STATE_HIDDEN.
+     */
```

Comment:
This `OnLayoutChangeListener` fixes the wrong bottom sheet initial position that causes this strange animation:

https://user-images.githubusercontent.com/2678039/177374383-724d1b4f-f079-400e-a31c-1caf509e7610.mp4